src/routes/productRouter.ts

```diff
@@ -105,35 +107,10 @@ router.get(
   productController.sellerGetProduct
 );
 
-router.post(
-  "/buyer-add-product-wishList/:id",
-  userAuthorization(["buyer"]),
-  isProductExistToWishlist,
-  productController.buyerAddProductToWishList
-);
-router.get(
-  "/buyer-view-whishlist-product",
```

Comment:
Typo error in **wishlist** => **whishlist**, and can be better if we call this: **buyer-view-whishlist-products** with ` s` because products in wishList are many.